docs/triage/README.md

```diff
@@ -341,7 +341,7 @@ During these meetings, we go through the following lists:
 
 * [P0](https://github.com/flutter/flutter/issues?q=is%3Aopen+label%3AP0+sort%3Aupdated-asc): all bugs should be assigned, and progress should be happening actively. There should be an update within the last week. If no progress is happening and owner cannot work on it immediately (e.g. they're on vacation, they're busy with their day job, family reasons, etc), find a new owner.
 * [Bugs flagged for additional triage](https://github.com/flutter/flutter/issues?q=is%3Aopen+label%3A%22will+need+additional+triage%22+sort%3Aupdated-asc+no%3Aassignee): figure out what should be done with the bug, then remove the `will need additional triage` label.
-* [flutter-pub-roller-bot](https://github.com/flutter/flutter/pulls/flutter-pub-roller-bot): check that the pub auto roller is chugging along.
+* [flutter-pub-roller-bot](https://github.com/flutter/flutter/pulls/flutter-pub-roller-bot): check that the pub auto roller is chugging along. If it has gotten trivially stuck, such as having a merge conflict, close the PR so that it can open a new one. If it is non-trivially stuck, <insert awesome advice here>.
```

Comment:
@christopherfujino do you know who can advise on this doc suggestion?

Reply:
I'd say file an infra ticket queue ticket: https://github.com/flutter/flutter/blob/master/docs/infra/Infra-Ticket-Queue.md

Reply:
Thanks. Filed https://github.com/flutter/flutter/issues/155723.

Reply:
Oops, sorry, I meant that we should advise the triager to file a ticket. I think at the point the auto-roller is broken, a human will need to investigate.

Reply:
OH! 🤣  Gotcha.

I will update this tomorrow. Hahaha